### ISSUE TITLE
Feature request: Codex localization matches the IDE localization settings

### ISSUE THREAD

**Issue by SoftFour** [DelphiWorlds/Codex#41]
Hi Dave,

I'm using the great Codex tool under my RadStudio IDE in Germany, but with English localization in the IDE.
This means, all my IDE menu's were in usually in English.

When I use Codex, I notice that their menu's were translated to German, which looks a little odd with the mixed languages.
It seems IDE translated to German
![grafik](https://github.com/user-attachments/assets/a538ea09-a800-4d20-a854-b7de8283c828)
![grafik](https://github.com/user-attachments/assets/4b785eb7-7f24-45ef-ae84-3c3898bb0d32)

while other pages stay still in English
![grafik](https://github.com/user-attachments/assets/38e5bea7-123d-428b-b139-cba11f0c1c4a)

I assume you choose the PC'S local language for the translation selection,
would it be possible to use the IDE language selection instead, so that the menu text language will match all other IDE text language?

**Comment by DelphiWorlds** [comment]
Yes, it would make sense for the Codex translations be based on the language that was used for Delphi. Just wondering whether this should be a configurable option? To me the answer would be: "no"

**Comment by SoftFour** [comment]
Perhaps this would be possible via registry
![grafik](https://github.com/user-attachments/assets/89d1671d-827e-4725-b71b-1f18a0e1bf01)

or the locales

![grafik](https://github.com/user-attachments/assets/98b7c8ab-3dc6-4cc8-aff2-e848f5296688)

Not sure if this can be reached via IOTA, like that

`uses
  ToolsAPI, System.Win.Registry, Winapi.Windows, System.SysUtils;

function GetIDELanguage: string;
var
  BaseRegistryKey: string;
  Registry: TRegistry;
begin
  Result := '';
  // Zugriff auf das IOTAServices-Interface, um den Basis-Registrierungsschlüssel zu erhalten
  BaseRegistryKey := (BorlandIDEServices as IOTAServices).GetBaseRegistryKey;
  Registry := TRegistry.Create(KEY_READ);
  try
    Registry.RootKey := HKEY_CURRENT_USER;
    // Öffnen des Basis-Registrierungspfads
    if Registry.OpenKeyReadOnly(BaseRegistryKey) then
    begin
      if Registry.ValueExists('InstallLanguage') then
      begin
        Result := Registry.ReadString('InstallLanguage');
        Exit;
      end;
    end;
    // Falls 'InstallLanguage' nicht vorhanden ist, 'Localization\Language' prüfen
    if Registry.OpenKeyReadOnly(BaseRegistryKey + '\Localization') then
    begin
      if Registry.ValueExists('Language') then
        Result := Registry.ReadString('Language');
    end;
  finally
    Registry.Free;
  end;
end;
`

It seems to be "InstallLanguage" in newer IDE and "Language" in older,
but I have not tested this code.
